pkg/deployment/list.go

```diff
@@ -56,3 +56,45 @@ func List(apiClient *clients.Settings, nsname string, options ...metaV1.ListOpti
 
 	return deploymentObjects, nil
 }
+
+// ListInAllNamespaces returns deployment inventory in the all the namespaces.
+func ListInAllNamespaces(apiClient *clients.Settings, options ...metaV1.ListOptions) ([]*Builder, error) {
+	passedOptions := metaV1.ListOptions{}
+	logMessage := "Listing deployments in all namespaces"
+
+	if len(options) > 1 {
+		glog.V(100).Infof("'options' parameter must be empty or single-valued")
```

Comment:
```suggestion
		glog.V(100).Infof("'options' parameter must be either empty or single-valued")
```

Reply:
This is just a copy-paste of the same message we use in all of our list functions. I don't think it matters much since it is the same everywhere else in our code